### ISSUE TITLE
Mejorar la experiencia al cambiar de artículo

### ISSUE THREAD

**Issue by Kevin-Arias-Lizarazo** [midudev/midu.dev#148]
![Captura de pantalla 2024-03-27 111608](https://github.com/midudev/midu.dev/assets/105768839/d09dfe9e-c404-427c-a1d8-1e0cd2ed3943)

modifica el anchor de las tarjetas que salen al final donde muestras otros artículos relacionados agregando esta sentencia al final '/#logo' lo cual seleccionara el logo de tu página y de esta forma al cambiar de articulo quedaras en la parte superior.

**Comment by aniketkumar7** [comment]
¿Quiere actualizar los enlaces de las tarjetas agregando /#logo en cursor.html ?

**Comment by Kevin-Arias-Lizarazo** [comment]
efectivamente, porque al darle clic te deja en la misma parte desde la que presionaste y se supone que el contenido que cambia esta arriba entonces tienes que moverte hacia arriba para poder leer el contenido que querías ver
